internal/report/report.go

```diff
@@ -1070,6 +1078,39 @@ func reportLabels(rpt *Report, g *graph.Graph, origCount, droppedNodes, droppedE
 		flatSum = flatSum + n.FlatValue()
 	}
 
+	if rpt.options.Focus != "" {
+		label = append(label, fmt.Sprintf("Using only samples with locations matching %s",
+			rpt.options.Focus))
+	}
+	if rpt.options.Ignore != "" {
+		label = append(label, fmt.Sprintf("Removed profile samples with locations matching %s",
```

Comment:
The filtering verb here is "ignore", so use that in the text? You do for tagignore below. Please make it consistent. In other places too.